alerts/UA00001.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dateRaised": 20200929,
+  "affectedFirmware": ["all"],
+  "hardwareLimited": [],
```

Comment:
This will only affect ChibiOS-based boards.  Maybe wedge it into `description`?

Reply:
STM32 rather than ChibiOS

Reply:
Added in STM32